server/package.json

```diff
@@ -13,6 +13,7 @@
     "@tailwindcss/line-clamp": "0.4.4",
     "@tsconfig/recommended": "1.0.6",
     "@types/cleave.js": "1.4.12",
+    "@types/dompurify": "^3.0.5",
```

Comment:
Great catch, I didn't notice that they didn't match. The internet says it's pretty normal for types packages to not match the packages they're created for, since types packages are generally "best effort". It sounds like it's ok for them to mismatch as long as we're aware of it. `cleave.js` also doesn't match it's types package.

And yep, this is basically owasp for javascript. I probably should have implemented a frontend sanitizer from the get go. `markdown-it` does not already sanitize. It does have a plugin for sanitizing that you can install, but `dompurify` is more popular and better maintained, so I think that's a better choice.

Reply:
I removed the `^` from dompurify and markdown-it